Druid/DRZ3ConstraintSolver.class.st

```diff
@@ -47,81 +101,216 @@ DRZ3ConstraintSolver >> isSatisfiable: aDRPiNode [
 	retval := solver check.
 
 	solver release.
-	^ retval
+	^ retval ifNil: [ true ]
+]
+
+{ #category : 'as yet unclassified' }
+DRZ3ConstraintSolver >> nextFreeVarOfType: type ifUnsupported: aBlock [
+
+	| varName converter |
+	converter := self getZ3ValueConverterFor: type ifUnsupported: aBlock.
+	
+	freeVarCounter := freeVarCounter + 1.
+	varName := 'z3tmp_' , freeVarCounter asString.
+	
+	^ converter value: varName.
+	
 ]
 
 { #category : 'as yet unclassified' }
```

Comment:
x (ok, this is not this PR, it got introduced before, but in one of the many previous ones ;))